doc/BUILDSERVER.md

```diff
@@ -193,6 +248,124 @@ verified or added to build Elektra correctly:
 - For Build Configuration you want to specify `by Jenkinsfile` and add the
   script path: `scripts/jenkins/Jenkinsfile`.
 
+### Jenkins libelektra-release Configuration
+
+The `libelektra-release` job is a normal pipeline job.
+
+Most of the default settings can be used.
+
+- Under General you need to enable `Permission to Copy Artifact` and set the field
+  to `*`.
+- For Build Configuration you want to specify `by Jenkinsfile` and add the
+  script path: `scripts/jenkins/Jenkinsfile.release`.
+
+### General Set-Up of a New Pipeline
+
+We will cover how to set-up a pipeline that uses a different git repository than
+this `libelektra` repository.
+
+Most of the default settings can be used.
+
+- If a private repository were to be used with a GitHub type job,
+  a credentials pair of `elektrabot` must be selected to access this repository.
+- For Build Configuration you want to specify `by Jenkinsfile` and add the
+  script path to the Jenkinsfile.
+
+We first start by including the shared library as described above in _Jenkins Shared Library_.
```

Comment:
Maybe move "Jenkins Shared Library" down here, it does not seem to be needed before?